run/run.py

```diff
@@ -218,7 +215,7 @@ def main(args):
         '--log-raw', LOG_FILEPATH,
         '--log-wptreport', REPORT_FILEPATH,
         '--log-mach=-',
-        '--processes=24', # FIXME remove insane mode option
+        '--processes=2', # FIXME remove insane mode option
```

Comment:
While working with this PR and testing it on individual smaller directories the numbers have been the same as expected, however I haven't done a larger scale analysis of whether increased concurrency leads to increased flakiness. It's probably best to lock it down to 1 until we have the infrastructure in place to detect an increase in flakiness.